README.md

```
@@ -1 +1,262 @@
+- Docker & Docker Compose
+- Make (optional, but recommended)
+
+### Setup & Run
+
+1. **Clone the repository**
+6. **Access the application**
+- [API](http://localhost:8000/)
+- [Health Check](http://localhost:8000/health/)
+- [API Documentation (Swagger)](http://localhost:8000/api/docs/)
+- [API Documentation (ReDoc)](http://localhost:8000/api/redoc/)
+- [Admin Panel](http://localhost:8000/admin/)
```

markdownlint-cli2 (0.18.1)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)